dockerfiles/compilation/java-8/compile.sh

```diff
@@ -47,9 +50,9 @@ git checkout $SHA1
 # Compilation
 
 if [ "$SKIPTESTS" = "skipTests" ]; then
-   mvn package -DskipTests
+   mvn package -DskipTests $MVN_ADDITIONAL_ARG_LINE
```

Comment:
I fear you don't handle spaces in your parameters with this syntax. If I remember correctly you should write `"${MVN_ADDITIONAL_ARG_LINE}"` instead

Reply:
Nice to know!